### DISCUSSION TITLE
Feature: Store tags in comicinfo.xml

### DISCUSSION THREAD

**Discussion by flaser** [maforget/ComicRackCE#37] [Ideas]
This is probably the one feature I find Comicrack lacking, even though some releases already supply this in some form (some releases use YAML, others XML).

IIRC, this plugin was supposed to enable saving-to/loading-from the xml, but I personally couldn't quite get it to work:
https://www.reddit.com/r/comicrackusers/comments/vjxyus/storeplus_02_bug_fix/

Note - this feature would likely necessitate an update to the schema:
https://gist.github.com/vaemendis/9f3ed374f215532d12bda3e812a130e6

**Comment by maforget** [comment]
Did you check the changelog before doing your request, or check the current build? Already done 3 weeks ago. 
https://github.com/maforget/ComicRackCE/commit/51d1162031c9b2e0dc63a0b00ce5695489441db8

Also ComicRack doesn't follow a schema, it just serialize it's class to an XML format. Other program need one, to make sure that the XML is compatible.

**Reply by flaser** [reply]
Awesome! Guess it's time for me to switch.

On a related note, could publishing schema based on the class be feasible?
I've been playing around editing ComicInfo.xml with scripts and it *is* useful to validate the result.

**Reply by maforget** [reply]
I just exported it with the xsd tool that comes with the framework using this command: `xsd.exe ComicRack.Engine.dll /t:ComicInfo`. Anyone can export it with their files. 

Don't know how exact it is, but looks ok, definitely has some error fixed from the other version , like Manga enumeration doesn't use `YesNo`.

[ComicRack.xsd.txt](https://github.com/maforget/ComicRackCE/files/14324206/ComicRack.xsd.txt)

**Reply by maforget** [reply]
Exported another with the PageType enum on it. The way the code is done it still uses it, but is not used for serializing, so the previous xsd only shows it as a string. At the end the result are the same though.

[ComicInfo.xsd.txt](https://github.com/maforget/ComicRackCE/files/14337713/ComicInfo.xsd.txt)
